public/js/components/BaseCard.vue

```diff
@@ -113,6 +113,16 @@ export default {
     };
   },
 
+  watch: {
+    draggingWithMouse: function(dragging) {
+      if (dragging) {
+        document.body.classList.add("dragging");
```

Comment:
this is much more elegant than what I implemented in the past to prevent selection. Thank you 🙏🏻